packages/stark-build/typings/index.d.ts

```diff
@@ -0,0 +1,3 @@
+/* tslint:disable:no-import-side-effect */
+import "./environment";
```

Comment:
Previously in Stark, we had
/* tslint:disable:no-reference */
/// <reference path="custom/environment.d.ts" />
/* tslint:enable */

Why do we use "import" here and not "reference" or,  maybe, "export" ?

Reply:
Inded, but in facta we were doing it wrong in Stark 9 :s 
When we need to reference typings from a path then the "import" syntax is preferred.

See https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html

Reply:
Okido 😊 
Thank you for the explanation